src/App.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+import { Header } from './components/Header';
+import { InputTodo } from './components/InputTodo';
+import { TodoList } from './components/TodoList';
+import { NavBar } from './components/NavBar';
```

Comment:
- [ ] kindly make this `Navbar` useable as you needed this in order to link different pages 👍 😉

Reply:
I think I made it usable, perhaps you can be more clear on what you mean